holoviews/plotting/renderer.py

```diff
@@ -187,6 +203,39 @@ def html(self, obj, fmt=None, css=None):
         return tag.format(src=src, mime_type=mime_type, css=css)
 
 
+    def static_html(self, obj, fmt=None, template=None):
+        """
+        Generates a static HTML with the rendered object in
+        the supplied format. Allows supplying a template
+        formatting string with fields to interpolate 'js',
+        'css' and the main 'html'.
+        """
+        cls_type = type(self)
+        renderers = (cls_type,) + cls_type.__bases__
```

Comment:
I'm not sure I understand the need for this approach. Why not use this sort of pattern?

```python
class BaseClass(object):
       js_dependencies = {'jquery':'url1'}

class Subclass(BaseClass):
       js_dependencies = dict(BaseClass.js_dependencies, **{'d3':'url2'})
```

Then you could use ``self.js_dependencies``, ``self.css_dependencies`` etc and you'll be able to get all the dependencies directly from the class attribute without iterating through the class hierarchy like this. Maybe I am missing something here?